src/AppBundle/Api/Normalizer.php

```diff
@@ -42,10 +42,13 @@ public function normalizeUser(User $user)
         }
 
         return [
-            'reference'   => $user->getReference(),
-            'firstName'   => $user->getFirstName(),
-            'lastName'    => $user->getLastName(),
-            'loginName'   => $user->getLoginName(),
+            'reference' => $user->getReference(),
+            'firstName' => $user->getFirstName(),
+            'lastName' => $user->getLastName(),
+            'displayName' => $user->getDisplayName(),
+            'loginName' => $user->getLoginName(),
+            'emailAddress' => $user->getEmailAddress(),
+            'avatarUrl' => $user->getAvatarUrl(),
             'annotations' => $annotations,
```

Comment:
Ow, in dit geval gaat het om normalisatie richting de API, de vraag is nu dus of de API nog wel `key` \ `value`  annotations accepteert?